unity/Assets/Scripts/Unity/UnityMap.cs

```diff
@@ -142,12 +146,12 @@ private void OnTilePlaced(object sender, TileMapEventArgs<ITile> e)
                     break;
             }
 
-            Instantiate(
+            var newTile = Instantiate(
                     template,
                     CoordinateToPoint(tile.Coordinate),
                     Quaternion.Euler(-90, 0, 0)
                 );
-
+            newTile.transform.parent = this.transform;
```

Comment:
what does this exactly do? This seems to be unity related but I cant quite find anything on it

Reply:
Remember when we had every single tile at the root of the object tree in unity? This makes new tiles children of the map - should any further bugs like the one in the last meeting rise up, I'm sure, we're going to be happy about some structure in the object tree.